### PR TITLE
[WIPTEST] Angular select by value

### PR DESCRIPTION
**Pull request by lehinevych**
Fix for angular select by value.

{{pytest: cfme/tests/services/test_catalog_item.py cfme/tests/automate/test_common_methods.py cfme/tests/infrastructure/test_providers.py -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 68695d46083d292f66bd0ba41a7231034218e095:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/web_ui/__init__.py [hunk 1]
```diff
@@ -3349,7 +3349,7 @@ def select_by_value(self, value):
         index = options_map.index(value)
         if not self.is_open:
             self.open()
-        new_loc = self._loc + '/../div/ul/li[@data-original-index={}]'.format(index)
+        new_loc = self._loc + '/../div/ul/li[@data-original-index={}]/a'.format(index)
```
Comment (mfalesni) [inline cfme/web_ui/__init__.py]: I see the builds failed. Why the `/a` now? :)

Comment (lehinevych) [inline cfme/web_ui/__init__.py]: the select_by_values method is used in adding buttons group (Automate->Customization), 
and without /a we couldn't select the button image from drop down list.

Comment (mfalesni) [inline cfme/web_ui/__init__.py]: Right, the `a` should be in every angular select so it should work with all of them?

Comment (lehinevych) [inline cfme/web_ui/__init__.py]: I've just checked and angular select by value is only used for image buttons. I'll check the failed tests, if the cause is connected with `/a` fix.

**Comment by dajoRH** [comment]
Lint report for commit a4a4746ef7f74712da8eec4f579e09d5dd8bcc46:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d7ce2fc8e7e513c9a0e0f4e8e1dbaade29a7d0db:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request closed without merging**